introduction/support.md

```diff
@@ -176,7 +176,7 @@ information in your message. A great way to provide your hardware details is by
 [generating and submitting a Hardware Compatibility List (HCL)
 report](/doc/how-to-use-the-hcl/#generating-and-submitting-new-reports), then
 linking to it in your message. [Ask questions the smart
-way.](http://www.catb.org/esr/faqs/smart-questions.html)
+way.](https://www.catb.org/esr/faqs/smart-questions.html)
```

Comment:
It seems `www.catb.org` doesn't support HTTPS.

Reply:
As of December 2023, the site does support HTTPS, but the certificate is invalid.

It is issued for `animadance.org` which well, is what it is... That being said, I don't think linking to the HTTPS version of the site and triggering the warning is making anything worse. It is a healthy warning, and the HTTP version probably warrants a similar warning anyway. I'd personally proceed with the change, YMMV.